From: Kel Modderman <kel@otaku42.de>
Date: Fri, 5 Apr 2019 20:32:41 +0000
Subject: rollup of updates to insserv.8:

* fix section number for init(8), thanks Regid Ichira <regid23@yahoo.com>
  (Closes: #634150)
* mention when insserv should (not) be used and how update-rc.d is almost
  always the interface you want to use (Closes: #566352)
* use the package short summary in the manual page too, it mentions the
  keyword "dependency" (Closes: #609655)
===================================================================
---
 insserv.8.in | 17 ++++++++++++-----
 1 file changed, 12 insertions(+), 5 deletions(-)

diff --git a/insserv.8.in b/insserv.8.in
index e791468..66daf70 100644
--- a/insserv.8.in
+++ b/insserv.8.in
@@ -26,7 +26,7 @@
 .UC 8
 @@END_SUSE@@
 .SH NAME
-insserv \- Enable an installed system init script
+insserv \- boot sequence organizer using LSB init.d script dependency information
 .SH SYNOPSIS
 .\"
 .B insserv
@@ -66,8 +66,10 @@ insserv \- Enable an installed system init script
 @@END_SUSE@@
 .SH DESCRIPTION
 .B insserv
-enables an installed system init script (`boot script')
-by reading the comment header of the script, e.g.:
+is a low level tool used by
+.B update\-rc.d
+which enables an installed system
+init script (`boot script') by reading the comment header of the script, e.g.:
 .sp 1
 .in +1m
 .nf
@@ -88,7 +90,11 @@ by reading the comment header of the script, e.g.:
 .fi
 .in -1m
 .sp 1
-and calculating the dependencies between all scripts.
+and calculating the dependencies between all scripts. It is not recommended to
+execute insserv directly unless you know exactly what you're doing, doing so
+may render your boot system inoperable.
+.B update\-rc.d
+is the recommended interface for managing init scripts.
 @@BEGIN_SUSE@@
 Please note, that the
 .B Default\-Stop
@@ -467,7 +473,8 @@ with the help of
 .BR checkproc (8),
 .BR killproc (8),
 @@END_SUSE@@
-.BR startpar (8).
+.BR startpar (8),
+.BR update\-rc.d (8).
 .SH COPYRIGHT
 2000\(en2009 Werner Fink,
 .br
